micb_soa.m

```diff
@@ -105,9 +105,10 @@
 % /////////////////////////////////////////////////////////////////////////
 %% Instructions %%
 Screen('FillRect',w,bgcolor);
-DrawFormattedText(w,'In the following task, you will follow, with your eyes, an array of striped patches moving across the screen. On EVERY TRIAL, one of the patches will rotate slightly while moving with its neighbors. When prompted, you will have to click on the patch that rotated.\n\nWe are testing what conditions make that rotation harder or easier to see, so do not be surprised if you did not see any rotation. Just do your best and take a guess if you are unsure.\n\nThe task is easiest if you follow the ''+'' sign at that appears at the middle of the array, so follow that with your eyes on each trial.\n\n\nLet the experimenter know if you have any questions.\n\nPress SPACEBAR to continue.','center','center',[],wrapAt);
+DrawFormattedText(w,'In the following task, you will follow, with your eyes, an array of striped patches moving across the screen.\n\nOn EVERY TRIAL, one of the patches will rotate slightly while moving with its neighbors. When prompted, you will have to click on the patch that rotated.\n\nWe are testing what conditions make that rotation harder or easier to see, so do not be surprised if you did not see any rotation. Just do your best and take a guess if you are unsure.\n\nThe task is easiest if you follow the dot that appears at the middle of the array, so follow that with your eyes on each trial.\n\n\nLet the experimenter know if you have any questions.\n\nClick the mouse to continue.','center','center',[]);

```

Comment:
can you use the original code from open science foundation and make sure that our instructions are just a verbatim copy of there instructions, no sense in trying to reinvent the wheel,

Reply:
I copied the instructions from the original code to make those instructions. There were just some things we had to change because they are different in our task (e.g., we use a dot instead of a cross for fixation and ask them to use the mouse to advance to the next screen).

Does that mean we should try to match the number of trials? It differs depending on how many SOAs we use.